content/docs/2.1/scalers/rabbitmq-queue.md

```diff
@@ -32,6 +32,7 @@ triggers:
 - `queueName`: Name of the queue to read message from. Required.
 - `queueLength`: Queue length target for HPA. Default is 20. Optional.
 - `protocol`: Protocol to be used for communication. Either `http` or `amqp`. It should correspond with the `host` value.
+- `vhostName`: Vhost to use for the connection, overrides any vhost set in the connection string from `host`/`hostFromEnv`.
```

Comment:
Could you please add this parameter to the yaml example above (~ line 25) with a note that it is and optional one?

Reply:
Added.